source/install/install-ubuntu-1604-postgresql.rst

```diff
@@ -100,5 +100,9 @@ Assume that the IP address of this server is 10.10.10.1
       You might have to install the PostgreSQL client software to use the command.
 
   The PostgreSQL interactive terminal starts. To exit the PostgreSQL interactive terminal, type ``\q`` and press **Enter**.
+  
+13. (Optional) Postgres recommends the standard way to pass passwords is through a ``.pgpass`` file. It is a special file that HAS to be 0600 in permissions otherwise it doesnt work. One can also pass the location of the file using the ``PGPASS`` env variable.
```

Comment:
```suggestion
13. (Optional) Postgres recommends the standard way to pass passwords is through a ``.pgpass`` file. It is a special file which has to be ``0600`` in permissions otherwise it will not work. One can also pass the location of the file using the ``PGPASS`` env variable.
```